cmd/oras/internal/option/output.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package option
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+// Output option struct.
+type Output struct {
+	OutputDescriptor bool
+	Pretty           bool
+}
+
+// ApplyFlags applies flags to a command flag set.
+func (opts *Output) ApplyFlags(fs *pflag.FlagSet) {
+	fs.BoolVarP(&opts.OutputDescriptor, "descriptor", "", false, "output descriptor")
+	fs.BoolVarP(&opts.Pretty, "pretty", "", false, "output prettified content")
+}
+
+// OutputContent outputs the content to stdout
+func (opts *Output) OutputContent(content []byte) error {
```

Comment:
I think this func can be removed after adding the pretty option

Reply:
A second thought says no: we need a func like `(opts *Output) Print(w io.Writer, desc ocispec.Descriptor) error` here which
1) checks if `OutputDescriptor` is set and skip for false
2) marshall the desc
3) w.Write

This can be resused for all push and delete commands for manifest and blob